lib/scripts/controllers/options.js

```diff
@@ -285,6 +293,18 @@ export class OptionCtrl {
     }
   }
 
+  updatePlaybackOptions(){
+    var playbackOpts = JSON.stringify({
+      'volume': this.volume,
+      'rate': this.rate,
+      'voiceName': this.voiceName,
+      'pitch': this.pitch
+    });
+    this.save({
```

Comment:
Could you add a comment in the code about this? Then, if we ever start seeing too many messages, we will know why.